drivers/tmp006/tmp006.c

```diff
@@ -226,9 +227,26 @@ void tmp006_convert(int16_t rawv, int16_t rawt,  float *tamb, float *tobj)
 
 int tmp006_read_temperature(const tmp006_t *dev, int16_t *ta, int16_t *to)
 {
+    uint8_t drdy;
+
+#if USE_RAW_VALUES
```

Comment:
Sorry to comment on something that I requested, but could you change the name to TMP006_USE_RAW_VALUES or TMP006_RAW_VALUES
This makes sure that if you specify CFLAGS=-DTMP006_RAW_VALUES in your Makefile/command line, you will not accidentally change some other behavior of the system.